packages/client/graph-scaffolder/src/core/renderer.ts

```diff
@@ -41,7 +41,7 @@ export const pathFn = d3
 export abstract class Renderer<V, E> extends EventEmitter {
 	options: Options;
 
-	parentMap: Map<string, INode<V>>;
```

Comment:
@mwdchang is this change fine?

Reply:
`parentMap` is used for grouping nodes together - what problem is this causing?